Xamarin.Forms.Platform.UAP/MasterDetailControl.cs

```diff
@@ -143,7 +143,7 @@ public Windows.Foundation.Size MasterSize
 
 				// On first load, the _commandBar will still occupy space by the time this is called.
 				// Check ShouldShowToolbar to make sure the _commandBar will still be there on render.
-				if (_firstLoad && _commandBar != null && ShouldShowToolbar)
+				if (_firstLoad && height != 0 && _commandBar != null && ShouldShowToolbar)
```

Comment:
If `height` is zero, then the `_firstLoad` flag won't ever be set to `false`. 
Instead of checking for a zero height here, could we just constrain height to a minimum of zero on line 157 below?